modules/core/classes/GalleryRepositoryIndex.class

```diff
@@ -57,6 +57,9 @@ class GalleryRepositoryIndex {
      */
     var $_utilities;
 
+    /* deprecated dynamic properties in php 8.2 */
+    var $utilities;
```

Comment:
I don't think this is needed? (looks like `$_utilities` above is used everywhere)

Reply:
Without the variable declaration unit tests 1665 and 1666 fails with the deprecation warning from php 8.2